tracer/test/snapshots/Rasp.AspNetCore5.SqlI_url=-Iast-ExecuteQueryFromBodyQueryData_exploit=SqlI_body={-UserName-- -' or '1'='1-}.verified.txt

```diff
@@ -48,6 +48,7 @@
       language: dotnet,
       runtime-id: Guid_1,
       span.kind: server,
+      _dd.appsec.fp.http.endpoint: http-post-a13f66cb--27691e7f,
```

Comment:
Are these values definitely stable?

Reply:
Yes, they are (or yes, they are supposed to)